Correcting status code 409 related error information. (#569)
camaraproject/Commonalities: documentation/CAMARA-API-Design-Guide.md

|       400        |   `{{SPECIFIC_CODE}}`   | `{{SPECIFIC_CODE_MESSAGE}}`                                         | Specific Syntax Exception regarding a field that is relevant in the context of the API (e.g. format of an amount)               |
|       400        |     `OUT_OF_RANGE`      | Client specified an invalid range.                                  | Specific Syntax Exception used when a given field has a pre-defined range or a invalid filter criteria combination is requested |
|       403        |   `PERMISSION_DENIED`   | Client does not have sufficient permissions to perform this action. | OAuth2 token access does not have the required scope or when the user fails operational security                                |
|       403        | `INVALID_TOKEN_CONTEXT` | `{{field}}` is not consistent with access token.                    | Reflect some inconsistency between information in some field of the API and the related OAuth2 Token. This error SHOULD be used only when the scope of the API allows it to explicitly confirm whether or not the supplied identity matches that bound to the Three-Legged Access Token.                             |

> |       409        |        `ABORTED`        | Resource is being modified by another operation. Please retry.      | Generic concurrency conflict (Race condition)                                                                                   |

Reviewer: ```suggestion
|       409        |        `ABORTED`        | Resource is being modified by another operation. Please retry.      | Concurrency of processes of the same nature/scope                                                                                   |

```

> |       409        |    `ALREADY_EXISTS`     | The resource that a client tried to create already exists.          | A resource with the same identifying characteristics already exists                                                             |

Reviewer: ```suggestion
|       409        |    `ALREADY_EXISTS`     | The resource that a client tried to create already exists.          | Trying to create an existing resource                                                             |

```

> |       409        |       `CONFLICT`        | A specified resource duplicate entry found.                         | Duplication of an existing (**Deprecated with Commonalities verion X**)                                                         |

Reviewer: To be removed for alignment

|       403        |   `PERMISSION_DENIED`   | Client does not have sufficient permissions to perform this action. | OAuth2 token access does not have the required scope or when the user fails operational security                                |
|       403        | `INVALID_TOKEN_CONTEXT` | `{{field}}` is not consistent with access token.                    | Reflect some inconsistency between information in some field of the API and the related OAuth2 Token. This error SHOULD be used only when the scope of the API allows it to explicitly confirm whether or not the supplied identity matches that bound to the Three-Legged Access Token.                             |
|       409        |        `ABORTED`        | Resource is being modified by another operation. Please retry.      | Concurrency of processes of the same nature/scope                                                                                   |

|       409        |    `ALREADY_EXISTS`     | The resource that a client tried to create already exists.          | Trying to create an existing resource                                                             |

> |       409        |       `CONFLICT`        | A specified resource duplicate entry found.                         | Duplication of an existing resource (**Deprecated with Commonalities X**)                                                        |

Reviewer 2: Aligning with CAMARA_common.yaml text:
```suggestion
|       409        |       `CONFLICT`        | A specified resource duplicate entry found.                         | Duplication of an existing resource (**This Error Code is DEPRECATED**)                                                        |

```

|       400        |   `{{SPECIFIC_CODE}}`   | `{{SPECIFIC_CODE_MESSAGE}}`                                         | Specific Syntax Exception regarding a field that is relevant in the context of the API (e.g. format of an amount)               |
|       400        |     `OUT_OF_RANGE`      | Client specified an invalid range.                                  | Specific Syntax Exception used when a given field has a pre-defined range or a invalid filter criteria combination is requested |
|       403        |   `PERMISSION_DENIED`   | Client does not have sufficient permissions to perform this action. | OAuth2 token access does not have the required scope or when the user fails operational security                                |
|       403        | `INVALID_TOKEN_CONTEXT` | `{{field}}` is not consistent with access token.                    | Reflect some inconsistency between information in some field of the API and the related OAuth2 Token. This error SHOULD be used only when the scope of the API allows it to explicitly confirm whether or not the supplied identity matches that bound to the Three-Legged Access Token.                             |

> |       409        |        `ABORTED`        | Resource is being modified by another operation. Please retry.      | Concurrency of processes of the same nature/scope                                                                                   |

Reviewer 3: > @Kevsy do you see any potential enhancements of the description in the last column?

@rartych good point, and yes: concurrent processes of the same nature/scope are not always a problem (i.e. two read operations). I suggest to change to "The resource is undergoing modification by another process"

> |       409        |        `ABORTED`        | Resource is being modified by another operation. Please retry.      | Concurrency of processes of the same nature/scope                                                                                   |

Author: Fixed.
